packages/web/public/static/locales/de/settings.json

```diff
@@ -31,7 +31,8 @@
       "cityLabel": "Stadt (optional)",
       "cityPlaceholder": "In welcher Stadt lebst du?",
       "countryLabel": "Land (optional)",
-      "countryPlaceholder": "In welchem Land lebst du?"
+      "countryPlaceholder": "In welchem Land lebst du?",
+      "updateHandleWarning": "Warning: updating your handle will break any existing links you have shared to your profile. If you continue, we recommend you update those links to have your new handle"
```

Comment:
@LindasLinguas there's a new string here to be translated whenever you have time 😄🙏🏼

Reply:
```suggestion
      "updateHandleWarning": "Warnung: Das Ändern des Benutzernamens führt dazu, dass die alten Links zu deinem Profil, die du bereits geteilt hast, nicht mehr funktionieren werden. Wenn du deinen Benutzernamen trotzdem ändern möchtest, empfehlen wir diese Links zu aktualisieren."
```